test/contract/EasyAuction.spec.ts

```diff
@@ -378,6 +395,69 @@ describe("EasyAuction", async () => {
         balanceBeforeOrderPlacement.sub(transferredbiddingTokenAmount),
       );
     });
+    it("places a new order and checks that allowListing prevents tx", async () => {
+      const AllowListManger = await ethers.getContractFactory(
+        "AllowListOffChainManaged",
+      );
+
+      const allowListManager = await AllowListManger.deploy();
+      const {
+        auctioningToken,
+        biddingToken,
+      } = await createTokensAndMintAndApprove(
+        easyAuction,
+        [user_1, user_2],
+        hre,
+      );
+      const auctionId: BigNumber = await sendTxAndGetReturnValue(
+        easyAuction,
+        "initiateAuction(address,address,uint256,uint256,uint96,uint96,uint256,uint256,bool,address)",
+        auctioningToken.address,
+        biddingToken.address,
+        60 * 60,
+        60 * 60,
+        ethers.utils.parseEther("1"),
+        ethers.utils.parseEther("1"),
+        1,
+        0,
+        false,
+        allowListManager.address,
+      );
+
+      const { chainId } = await ethers.provider.getNetwork();
+      const testDomain = domain(chainId, allowListManager.address);
+      const auctioneerMessage = ethers.utils.keccak256(
+        ethers.utils.defaultAbiCoder.encode(
+          ["bytes32", "address", "uint256"],
+          [
+            ethers.utils._TypedDataEncoder.hashDomain(testDomain),
+            user_2.address,
+            auctionId,
+          ],
+        ),
+      );
+      // Signature will come from a wrong user: user_2 != owner of allowListManger;
+      const auctioneerSignature = await user_2.signMessage(
+        ethers.utils.arrayify(auctioneerMessage),
+      );
+      const sig = ethers.utils.splitSignature(auctioneerSignature);
+      const auctioneerSignatureEncoded = ethers.utils.defaultAbiCoder.encode(
+        ["uint8", "bytes32", "bytes32"],
+        [sig.v, sig.r, sig.s],
+      );
+
+      const sellAmount = ethers.utils.parseEther("1").add(1);
+      const buyAmount = ethers.utils.parseEther("1");
+      await expect(
+        easyAuction.placeSellOrders(
+          auctionId,
+          [buyAmount, buyAmount],
+          [sellAmount, sellAmount.add(1)],
+          [queueStartElement, queueStartElement],
+          auctioneerSignatureEncoded,
+        ),
+      ).to.be.revertedWith("user not allowed to call function");
+    });
```

Comment:
I'd move this test to `test/contract/AllowListManager.spec.ts`, since it relies on the logic of `AllowListOffChainManaged`, and use a mock for `allowListManager`.
I'd also add a different "positive" test with a mock that shows that the order is correctly placed. (So two tests more in total.)